examples/advanced.html

```diff
@@ -12,6 +12,19 @@
       media-loading-indicator {
         --media-loading-icon-width: 100px;
       }
+
+      media-airplay-button[media-airplay-unavailable] {
```

Comment:
I'm wondering if we should just go ahead and build this into the control bar. But that may open up other issues so it can wait for a later PR either way.